### PR TITLE
Document node capacity.

### LINKED ISSUE #7265: Feature request: Ability for kubelet to express different node capacity
Today, Kubelet reports all resources as Node capacity to Apiserver.
Looking for ability to reserve CPU & Memory for system and other tasks not managed by kubernetes and report the rest as Node capacity to Apiserver.

### PR DESCRIPTION
**Pull request by erictune**
Fixes #7265 in a hacky way.

### CODE REVIEW

**Comment by erictune** [comment]
@vmarmol @dchen1107 
This is suggested a way to reserve capacity for system services on the node.
Should I publish this, or will it conflict with your plans?

File: docs/node.md [hunk 1]
```diff
@@ -149,5 +149,36 @@ unschedulable, run this command:
...
+capacity when adding a node.
+
+The kubernetes scheduler ensures that there are enough resources for all the pods on a node.  It
+checks that the sum of the limits of containers on the node is less than the node capacity.  It
+includes all containers started by kubelet, but not containers started directly by docker, nor
+processes not in containers.  
+
+If you want to explicitly reserve resources for non-Pod processes, you can create a placeholder
+pod.  Use the following template:
+```
```
Comment (mbforbes) [inline docs/node.md]:
```yaml ?

**Comment by mbforbes** [comment]
(As oncall merging docs changes, I have the following request: please add an `ok-to-merge` label as well as the standard LGTM once this is LGTM—I'm perusing through that list for merges. Many thanks to you!)

File: docs/node.md (hunk 1 above)
Comment (erictune) [inline docs/node.md]: fixed.

**Comment by vmarmol** [comment]
LGTM, thanks @erictune!

**Pull request merged**